omod/src/main/java/org/openmrs/module/reportingrest/web/resource/MappedReportDefinitionConverter.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License

```

Comment:
This is the wrong license. We now use MPLv2. See http://openmrs.org/license/
(You can probably copy this from some existing file elsewhere into your IntelliJ settings.)